config/default.yml

```diff
@@ -721,6 +721,16 @@ RSpec/FactoryBot/AttributeDefinedStatically:
   VersionChanged: '2.0'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FactoryBot/AttributeDefinedStatically
 
+RSpec/FactoryBot/ConsistentParenthesesStyle:
+  Description: Use a consistent style for parentheses in factory bot calls.
+  Enabled: pending
+  EnforcedStyle: enforce_parentheses
+  SupportedStyles:
+    - enforce_parentheses
```

Comment:
For consistency with other Rubocop's cops I'd suggest to name this `require_parentheses`